### PR TITLE
[@labs/gen-wrapper-react] TestOutput links to monorepo for dependencies

### PR DESCRIPTION
**Pull request by taylor-vann**
This PR links `test-output` to the monorepo for the following dependencies:
- react
- react types
- lit
- lit-element
- lit-html

This avoids version conflicts during tests where we emulate a developer using `gen-wrapper-react`.

### CODE REVIEW

**Comment by justinfagnani** [comment]
Failures look like flakiness. I'll re-run

File: packages/labs/gen-wrapper-react/src/index.ts [hunk 1]
```diff
@@ -94,8 +94,8 @@ const packageJsonTemplate = (
       },
       peerDependencies: {
         // TODO(kschaaf): make react version(s) configurable?
-        react: '^17 || ^18',
-        '@types/react': '^17 || ^18',
```
Comment (kevinpschaaf) [inline packages/labs/gen-wrapper-react/src/index.ts:98]: Can these go back to 17 || 18 now? Or does it break if we do that? Seems best for the wrapper to support wider versions if possible.

**Review by kevinpschaaf — APPROVED** [review]
Changes LGTM, just one question about the versions.

File: packages/labs/gen-wrapper-react/src/index.ts (hunk 1 above)
Comment (taylor-vann) [reply packages/labs/gen-wrapper-react/src/index.ts:98]: let's try it out!

Comment (taylor-vann) [reply packages/labs/gen-wrapper-react/src/index.ts]: success!

**Pull request merged**